docs/content/docs/dev/table/functions/udfs.md

```diff
@@ -1172,6 +1173,153 @@ If you intend to implement or call functions in Python, please refer to the [Pyt
 
 {{< top >}}
 
+Asynchronous Table Functions
```

Comment:
you should add placeholder information into the Chinese docs.

Reply:
Done